cvat-core/src/api-implementation.ts

```diff
@@ -34,6 +34,10 @@ import QualitySettings from './quality-settings';
 import { FramesMetaData } from './frames';
 import AnalyticsReport from './analytics-report';
 
+function isPageSize(value: any) {
```

Comment:
Not sure it's right, other checking functions used accept `any`, and it makes sense to check `any`, because we want to validate an arbitrary passed value.

Reply:
This is what type we expect used to static linting.
Code changes what actually what passed in runtime.

If we pass any, we already do not see type errors by static analizer